static-build/qemu/Dockerfile

```diff
@@ -54,4 +54,7 @@ RUN PREFIX="${PREFIX}" /root/configure-hypervisor.sh -s kata-qemu | xargs ./conf
 
 RUN make -j$(nproc)
 RUN make install DESTDIR=/tmp/qemu-static
+# Remove u-boot firmware manually because it's not needed in kata,
+# has a lot of CVEs and there is no option in qemu to disable it
+RUN find /tmp/qemu-static -name "u-boot*" -exec rm -f {} \;
```

Comment:
There are a longer list of items we actually don't need from the build output.  I wonder if we should have a blacklist/whitelist?

Reply:
sounds good, let me check what files are needed